libraries/CAN/test/unit/src/framework/arduino/corelibs/Test_CAN_connected_node1.cpp

```diff
@@ -0,0 +1,44 @@
+// std includes
```

Comment:
You may even merge the related files under arduino, features and corelibs as everything is taking place in the same environment. This was different for the 3D magnetic library : the tests had to be independent of of Arduino functionality in order to be reusable for MTB. But here separating the directories does not make sense, all is taking place in Arduino.
This will also allow you to better test the code currently in the arduino/corelibs directory.

Reply:
Need a clearer stucture